content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -155,33 +162,41 @@ int CalculateTestSquare( const int number)
 
 TEST(MathTests, FastSquare_Integer_Squared1)
 {
+    // despite the name suggesting this verifies the math of computing a square
+    // this test trusts the function is correct, and only proves an identity property
     int tenSquared = Math::FastSquare(10);
     int result = Math::FastSquare(10);
     ASSERT_EQ(result, tenSquared);
 }
 
 TEST(MathTests, FastSquare_Integer_Squared2)
 {
+    // relies on a duplicate implementation of the production code which contains
+    // the same bug, and only helps prove the production bug was not changed
     int tenSquared = CalculateTestSquare(10);
     int result = Math::FastSquare(10);
     ASSERT_EQ(result, tenSquared);
 }
 
 TEST(MathTests, FastSquare_Integer_Squared3)
 {
+    // uses incorrectly computed squares from another file, and only helps
+    // prove the production bug was not changed
     int result = Math::FastSquare(10);
     ASSERT_EQ(result, pcs_ten);
 }
 
 TEST(MathTests, FastSquare_Integer_Squared4)
 {
+    // directly states that 10^2 = 20, which is incorrect but also easy to notice
```

Comment:
```suggestion
    // Verifies that 10^2 = 20, which is an incorrect expectation
```

Reply:
"easy to notice" is the point this example is trying to make. "directly states" is the reason it is easy to notice.